misc/audit-manifest.txt

```diff
@@ -0,0 +1,186 @@
+https://www.adobe.com/cc-shared/assets/sitemap.xml
+https://www.adobe.com/au/cc-shared/assets/sitemap.xml
+https://www.adobe.com/cn/cc-shared/assets/sitemap.xml
```

Comment:
where does the list of sitemaps coming from?

Reply:
i created it out of the locales we have in our code

Reply:
we should probably add a disclamer that these are not all Milo pages, wdyt?

Reply:
this manifest is here for somebody (me for now) to use. Nothing say this manifest is here to get everything :) (i wait for discussion with business for this)